frontend/src/ui/Input.tsx

```diff
@@ -68,11 +68,13 @@ export const Select = React.forwardRef<HTMLSelectElement, SelectProps>(
 
 type CopyableInputProps = JSX.IntrinsicElements["div"] & {
     value: string;
+    context: "direct-link" | "embed-code" | "oc-id";
```

Comment:
I thiiiink I would rather (especially with the comment above) just pass a `label: string` here :/